Octokit/Http/SimpleJsonSerializer.cs

```diff
@@ -122,8 +122,19 @@ public override object DeserializeObject(object value, Type type)
                             //dictionary does not contain enum value and has no custom attribute. So add it for future loops and return value
                             // remove '-' from values coming in to be able to enum utf-8
                             stringValue = RemoveHyphenAndUnderscore(stringValue);
-                            var parsed = Enum.Parse(type, stringValue, ignoreCase: true);
-                            _cachedEnums[type].Add(value, parsed);
+                            object parsed = new object();
+                            try
+                            {
+                                parsed = Enum.Parse(type, stringValue, ignoreCase: true);
```

Comment:
ah yeah sorry i didnt look at the diff context and see now that it's inside a non generic method etc, so the way you've done it is ok